common_features.mk

```
@@ -321,3 +321,10 @@ ifeq ($(strip $(SPLIT_KEYBOARD)), yes)
     endif
     COMMON_VPATH += $(QUANTUM_PATH)/split_common
 endif
+
+ifeq ($(strip $(OLED_DRIVER_ENABLE)), yes)
+    OPT_DEFS += -DOLED_DRIVER_ENABLE
+    COMMON_VPATH += $(DRIVER_PATH)/oled
+    SRC += i2c_master.c
```

Could this be changed to: 

```suggestion
    QUANTUM_LIB_SRC += i2c_master.c
```

This should still be added properly, but will prevent warning messages when multiple features add i2c_master.

---

k, fixed